spacewalk/admin/spacewalk.target

```diff
@@ -14,6 +14,7 @@ Requires=rhn-search.service
 Requires=cobblerd.service
 Requires=taskomatic.service
 Requires=spacewalk-wait-for-taskomatic.service
+Requires=salt-secrets-config.service
```

Comment:
Oh yes. We did not really maintain spacewalk.target as we use the file spacewalk.target.SUSE .
Please check if other changes we made there should be ported as well.

Btw: we made jabberd, osa-dispatcher "optional" and start them manual in `spacewalk-service` script.
I you have a salt only system, you do not what them.

Reply:
Ah, I was indeed confused about the optional. I will add the line-up between the two files to my to-do list. Maybe we can even drop the .SUSE version for a generic one.